Code/Engine/Foundation/FoundationPCH.cpp

```diff
@@ -121,11 +110,107 @@ EZ_STATICLINK_LIBRARY(Foundation)
   EZ_STATICLINK_REFERENCE(Foundation_Memory_Implementation_AllocatorWrapper);
   EZ_STATICLINK_REFERENCE(Foundation_Memory_Implementation_EndianHelper);
   EZ_STATICLINK_REFERENCE(Foundation_Memory_Implementation_FrameAllocator);
+  EZ_STATICLINK_REFERENCE(Foundation_Memory_Implementation_InstanceDataAllocator);
   EZ_STATICLINK_REFERENCE(Foundation_Memory_Implementation_MemoryTracker);
   EZ_STATICLINK_REFERENCE(Foundation_Memory_Implementation_MemoryUtils);
   EZ_STATICLINK_REFERENCE(Foundation_Memory_Implementation_PageAllocator);
-  EZ_STATICLINK_REFERENCE(Foundation_Memory_Policies_GuardedAllocation);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_Application_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_ConditionVariable_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_CrashHandler_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_DirectoryWatcher_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_EnvironmentVariableUtils_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_GuardedAllocation_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_MemoryMappedFile_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_MiniDumpUtils_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_OSFile_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_OSThread_Android);
+  EZ_STATICLINK_REFERENCE(Foundation_Platform_Android_PageAllocator_Android);
```

Comment:
All of these should not be neccessary. Maybe we need to teach the static link util to ignore platform specific files?

Reply:
In general this static link stuff is only required for things that are only available through reflection. As soon as you have methods / classes which are dllexport they always end up in the final binary. BEcause this is the case for all these platform specific things, the staticlink reference stuff here is just additional bloat without function.

Reply:
Deactivated for these files now.